src/css/_manager.scss

```diff
@@ -0,0 +1,78 @@
+.manage-main {
+  height: 100vh;
+  margin-top: 15vh;
+}
+
+.manage-search {
+  
+}
+
+.search-form {
+  width: 100%;
+  text-align: end;
+  margin: -2em;
+  padding-top: 1em;
+}
+
+.manage-search {
+  @extend .search-form;
+}
+
+.calendar {
```

Comment:
I like how you've basically set up what you'll need in the future. Good planning

Reply:
Yeah, it helps me personally to keep on track also makes it easy to add in a style so I can play with the layout in chrome dev tools styling